app/Domain/Menu/Templates/headMenu.blade.php

```diff
@@ -208,12 +208,14 @@ function toggleNotificationTabs(active) {
                 jQuery('.notificationHandler').on('click', function () {
                     jQuery.ajax(
                         {
-                            type: 'PATCH',
-                            url: leantime.appUrl + '/api/notifications',
+                            method: 'GET',
+                            url: '/api/notifications',
```

Comment:
We still need the appUrl otherwise subfolder installations will not work. Can you change the script section to be a module and then import instance-info.module.mjs
which contains appUrl